samples/DurableFunctions/README.md

```diff
@@ -206,9 +138,13 @@ docker build --platform linux --tag <DOCKER_ID>/azuredurablefunctionsimage:v1.0.
 
 In this example, replace \<DOCKER_ID\> with your Docker Hub account ID. When the command completes, you can run the new container locally.
 
-2\. To test the build, run the image in a local container using the [docker run](https://docs.docker.com/engine/reference/commandline/run/) command, with the adding the ports argument, -p 8080:80.
+2\. To test the build, run the image in a local container using the [docker run](https://docs.docker.com/engine/reference/commandline/run/) command, with the ports argument, -p 8080:80 and environment variables `AzureWebJobsStorage` and `SQLDB_Connection`: 
+
 ```sh
-docker run -p 8080:80 -it <DOCKER_ID>/azuredurablefunctionsimage:v1.0.0
+docker run -it <DOCKER_ID>/azuredurablefunctionsimage:v1.0.0 \
+-e AzureWebJobsStorage='Connection string' \
+-e SQLDB_Connection='Connection string' \
+-p 8080:80
```

Comment:
I think this command will fail if image name is not provided at the end of the command.